dev/ci/user-overlays/13952-mattam82-hints-modes.sh

```diff
@@ -0,0 +1,2 @@
+overlay coqhammer https://github.com/mattam82/coqhammer pr-13952 13952
+overlay rewriter https://github.com/mattam82/rewriter coq-13952 13952
```

Comment:
Should not be needed anymore since https://github.com/mit-plv/rewriter/pull/24 was merged

Reply:
Yep, I'll remove that in 13952 if not done already.